rpm.template/retroshare.spec

```diff
@@ -0,0 +1,636 @@
+Name:          retroshare
```

Comment:
should probably be renabed to retroshare-gui for conformity with other distributions